themes/codesandbox-dark.json

```diff
@@ -163,11 +163,6 @@
       "scope": ["constant"],
       "settings": { "foreground": "#7AD9FB" }
     },
-    {
-      "name": "invalid",
```

Comment:
We should avoid root-level scopes at all cost: there's no way to override them on a more specific level of tokens.